Eatery/Controllers/BRB/BRBAccountViewController.swift

```diff
@@ -0,0 +1,115 @@
+//
+//  BRBAccountViewController.swift
+//  Eatery
+//
+//  Created by William Ma on 9/1/19.
+//  Copyright © 2019 CUAppDev. All rights reserved.
+//
+
+import UIKit
+
+// MARK: -
```

Comment:
Is this intentional or did you forget to put something here?

Reply:
This was just a habit I got into over the summer. I'll get rid of them to follow the AppDev style guide